include/UnitSpinAmplitude.h

```diff
@@ -60,7 +69,7 @@ class UnitSpinAmplitude : public SpinAmplitude
     UnitSpinAmplitude(unsigned two_J, const SpinVector& two_j, unsigned l, unsigned two_s)
         : SpinAmplitude(two_J, two_j, l, two_s, equal_always)
     {
-        addAmplitude(0, SpinProjectionVector(two_j.size(), 0));
+        addAmplitude(0, SpinProjectionVector(two_j.size(), 0), true);
```

Comment:
For two-particle decays: create all necessary amplitudes: loop from -J to J; also choose correct daughter spin projections.
For more-than-two-body decays: throw if not everything is spin zero.